tests/rados/test_object_ops.py

```diff
@@ -0,0 +1,88 @@
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.parallel import parallel
+from ceph.rados.core_workflows import RadosOrchestrator
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    # CEPH-83571710
+    Test to verify parallel IOPs on an object in a pool
+    1. Create a replicated pool
+    2. Perform the below two tasks parallely:
+        - Use rados put to write 4MB data to an object 15 times with
+            initial offset 0 and incremental offset of 8MB from client node
+        - Use rados put to write 4MB data to an object 15 times with
+            initial offset 4MB and incremental offset of 8MB from installer node
+    3. Verify the ceph df stats and ensure the pool has only 1 object
+        and stored data is 120 MB
+    4. Delete the created pool
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    client_node = ceph_cluster.get_nodes(role="client")[0]
+    installer_node = ceph_cluster.get_nodes(role="installer")[0]
+
+    pool_name = "re_pool_parallel_io"
+    object_name = "obj_parallel_io"
+    log.info("Running test case to verify parallel I/O operations on an Object")
+    put_cmd = f"rados put -p {pool_name} {object_name} /mnt/sample_4M"
+
+    def rados_put_installer(installer_offset=4194304):
```

Comment:
Move this as a common method in ceph core. Also, take input as to how many objects should be created on the pool.

Reply:
Configure for varied size objects. divide the size of object, by 4 for no of loops to perform.

Reply:
Changes have been made, please review latest code.